core/src/main/java/fj/P2.java

```diff
@@ -1,8 +1,10 @@
 package fj;
 
 import static fj.Function.*;
-
+import static fj.data.optic.PLens.pLens;
```

Comment:
Lenses (optics) are way more important to functional programming, in a practical regard.